web/client/actions/geostory.js

```diff
@@ -146,3 +153,33 @@ export const updateCurrentPage = ({sectionId}) => ({
     type: UPDATE_CURRENT_PAGE,
     sectionId
 });
+
+/**
+ * sort cards in builder content list after drag & drop events
+ * @param {string} source source path that is being dragged
+ * @param {string} target target path in the story, where to place dragged item
+ * @param {string|number} position new position of the target drop
+ * @param {string} newId id of the new card that is created, it is temporary
```

Comment:
if i remove them i need to recreate paths inside the sortContentEpic of geostory , which is not so easy to do.

since we are using a weird way to update positions, which is:
- remove item
- add item
- update item id with id of removed item
those informations are pretty useful

Reply:
didn't get why you have to change the ID temporary.

Reply:
Going to add this to conversation the next review